lets-encrypt task failed to generate a certificate

New Streisand server on DigitalOcean.
Have a domain registered and pointed to the IP of the generated droplet.
After entering it, the task for Let's Encrypt cert fails, and the default cert was used instead.

### Expected behavior:

Task `[lets-encrypt : Request initial certificate from Let's Encrypt]` should have succeeded.

### Actual Behavior:

`FAILED! => `
```json
{
    "changed": true,
    "cmd": [
        "acmetool",
        "want",
        "my.doma.in"
    ],
    "delta": "0:00:01.067966",
    "end": "2018-05-11 14:20:18.914240",
    "msg": "non-zero return code",
    "rc": 1,
    "start": "2018-05-11 14:20:17.846274",
    "stderr": "20180511142018 [ERROR] acme.storageops: could not obtain authorization for my.doma.in: failed all combinations\n20180511142018 [ERROR] acme.storageops: Target(my.doma.in;https://acme-v01.api.letsencrypt.org/directory;0): failed to request certificate: failed all combinations\n20180511142018 [ERROR] acme.storageops: error while processing targets: the following errors occurred:\nerror satisfying Target(my.doma.in;https://acme-v01.api.letsencrypt.org/directory;0): failed all combinations\n20180511142018 [ERROR] acme.storageops: failed to reconcile: the following errors occurred:\nerror satisfying Target(my.doma.in;https://acme-v01.api.letsencrypt.org/directory;0): failed all combinations\n20180511142018 [CRITICAL] acmetool: fatal: reconcile: the following errors occurred:\nerror satisfying Target(my.doma.in;https://acme-v01.api.letsencrypt.org/directory;0): failed all combinations",
    "stderr_lines": [
        "20180511142018 [ERROR] acme.storageops: could not obtain authorization for my.doma.in: failed all combinations",
        "20180511142018 [ERROR] acme.storageops: Target(my.doma.in;https://acme-v01.api.letsencrypt.org/directory;0): failed to request certificate: failed all combinations",
        "20180511142018 [ERROR] acme.storageops: error while processing targets: the following errors occurred:",
        "error satisfying Target(my.doma.in;https://acme-v01.api.letsencrypt.org/directory;0): failed all combinations",
        "20180511142018 [ERROR] acme.storageops: failed to reconcile: the following errors occurred:",
        "error satisfying Target(my.doma.in;https://acme-v01.api.letsencrypt.org/directory;0): failed all combinations",
        "20180511142018 [CRITICAL] acmetool: fatal: reconcile: the following errors occurred:",
        "error satisfying Target(my.doma.in;https://acme-v01.api.letsencrypt.org/directory;0): failed all combinations"
    ],
    "stdout": "",
    "stdout_lines": []
}
```

### Steps to Reproduce:
1. Pick DigitalOcean as provider
2. Use custom SSH key
3. Set settings as written below
4. Pick DO server location, name, VPN key name
5. Use provided IP address for the `A` domain record (replaced in log with `my.doma.in`)
6. Feed that domain into lets-encrypt task prompt

*Result* - error from above during the server setup.

### Ansible Information

* Ansible version: 2.5.2
* Ansible system: Darwin
* Host OS: MacOSX
* Host OS version:  10.13.4
* Python interpreter: python
* Python version: 2.7.15

### Streisand Information

* Streisand Git revision: 3031d005010c6dea691ea2f96044592cfe40db04
* Streisand Git clone has untracked changes: no
* Genesis role: genesis-digitalocean
* Custom SSH key: True

### Enabled Roles

* Shadowsocks enabled:  True
* Wireguard enabled: True
* OpenVPN enabled: True
* stunnel enabled: True
* Tor enabled: False
* Openconnect enabled: False
* TinyProxy enabled: True
* SSH forward user enabled: True
* Configured number of VPN clients: 5

--- a/playbooks/roles/sslh/tasks/main.yml
+++ b/playbooks/roles/sslh/tasks/main.yml
@@ -2,6 +2,7 @@
 - name: Install sslh
   apt:
     name: sslh
+    install_recommends: false
 
 - name: Generate the sslh port multiplexer systemd defaults file
   template:

